meinberlin/apps/budgeting/templatetags/react_proposals_vote.py

```diff
@@ -29,17 +29,16 @@ def react_proposals_vote(context, module, proposal):
     if "voting_tokens" in request.session:
         module_key = str(module.id)
         if module_key in request.session["voting_tokens"]:
-            try:
-                token = VotingToken.objects.get(
-                    token=request.session["voting_tokens"][module_key], module=module
-                )
-                serializer = VotingTokenSerializer(token)
-                token_info = serializer.data
-                session_token_voted = TokenVote.objects.filter(
-                    token__pk=token.pk, content_type=proposal_ct, object_pk=proposal.pk
-                ).exists()
-            except VotingToken.DoesNotExist:
-                pass
+            token = VotingToken.get_voting_token_by_hash(
+                token_hash=request.session["voting_tokens"][module_key], module=module
+            )
+            if not token:
+                return
```

Comment:
what does this return do?

Reply:
I can also flip it to `if token:` if that's better to read

Reply:
yeah maybe, then its also more consistent (like in VotingTokenInfo).. thanks!